packages/theme/src/theme/_navbar.search.scss

```diff
@@ -10,25 +10,20 @@
 				content: '';
 				flex: 1 0 auto;
 			}
-
-			@media (min-width: $grid-float-breakpoint) {
-				padding-top: $navbar-padding-vertical;
-				padding-bottom: $navbar-padding-vertical;
-			}
 		}
 
 		.form-control {
-			padding-right: $navbar-search-btn-width;
```

Comment:
`$navbar-search-btn-width` is not used anymore. Could you delete it from _variables.scss ?